blog/2023-02-22-consensus.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Consensus Team Update
+slug: 2023-02-22-consensus
+authors: dnadales
+tags: [consensus]
+hide_table_of_contents: false
+---
+
+## High level summary
+
+## Workstreams 
+
+### UTxO HD Prototype 
+
+We got the results of the first system level benchmarks for UTxO HD. They seemed
+to indicate a significant regression in performance. After looking into the
+benchmark logs we found that the benchmark runs took ledger state snapshots too
+often, due to the default snapshotting policy depending on `k`, and `k` being so
+small in the benchmark runs. Therefore, the next step is to re-run the
+benchmarks with a snapshotting policy that more closely resembles the one from
+mainnet.
+
+At the same time, we continued refactoring and cleaning up the prototype. 
+
+Also, we prepared the `anti-diff` packages (`fingertree-rm`, `diff-containers`,
+`simple-semigroupoids`) for publication in CHaP. This involved setting up CI,
+documentation, and some refactoring. These packages are now published to
+[CHaP][chap]. Also we published `cardano-lmdb` and `cardano-lmdb-simple` to
+CHaP.
```

Comment:
```suggestion
Also, we published the `anti-diff` packages (`fingertree-rm`, `diff-containers`,
`simple-semigroupoids`) and the LMDB packages (renamed to 
`cardano-lmdb` and `cardano-lmdb-simple`)  in [CHaP][chap]. 
```

I saw too many times the word `CHaP`, Feel free to dismiss

Reply:
Good point. We should keep things simple, also in the prose :) I'll change this.